Makefile

```diff
@@ -36,37 +36,43 @@ setup:
 .PHONY: pipenv-install
 pipenv-install: lib/Pipfile
 	@# Runs pipenv install; doesn't update the Pipfile.lock.
-	cd lib; pipenv install --dev
+	cd lib; pipenv install --dev --pre
 
 .PHONY: pipenv-lock
 # Re-generate Pipfile.lock. This should be run when you update the Pipfile.
 pipenv-lock: lib/Pipfile
 	@# Regenerates Pipfile.lock and rebuilds the virtualenv. This is rather slow.
 # In CircleCI, dont generate Pipfile.lock This is only used for development.
 ifndef CIRCLECI
-	cd lib; rm -f Pipfile.lock; pipenv lock --dev && mv Pipfile.lock Pipfile.locks/$(PY_VERSION)
+	cd lib; rm -f Pipfile.lock; pipenv lock --dev --pre && mv Pipfile.lock Pipfile.locks/$(PY_VERSION)
 else
 	echo "Running in CircleCI, not generating requirements."
 endif
 	cd lib; rm -f Pipfile.lock; cp -f Pipfile.locks/$(PY_VERSION) Pipfile.lock
 ifndef CIRCLECI
 	# Dont update lockfile and install whatever is in lock.
-	cd lib; pipenv install --ignore-pipfile --dev
+	cd lib; pipenv install --ignore-pipfile --dev --pre
 else
-	cd lib; pipenv install --ignore-pipfile --dev --system
+	cd lib; pipenv install --ignore-pipfile --dev --system --pre
 endif
 
 .PHONY: pylint
 # Run Python linter.
 pylint:
 	# Linting
 	# (Ignore E402 since our Python2-compatibility imports break this lint rule.)
-	cd lib; \
-		flake8 \
-		--ignore=E402,E128 \
-		--exclude=streamlit/proto/*_pb2.py \
-		$(PYTHON_MODULES) \
-		tests/
+	# cd lib; \
+	# 	flake8 \
+	# 	--ignore=E402,E128 \
+	# 	--exclude=streamlit/proto/*_pb2.py \
+	# 	$(PYTHON_MODULES) \
+	# 	tests/
```

Comment:
I assume all code produced by `black` is PEP8-compliant. In which case we don't need `flake8` -- so why not just remove this?

Reply:
Removed.